plugins/reports/articles/ArticleReportPlugin.inc.php

```diff
@@ -60,232 +55,221 @@ function display($args, $request) {
-					//'editor_country' . $e => __('common.country') . " (" . __('user.role.editor') . " $e)",
-					//'editor_affiliation' . $e => __('user.affiliation') . " (" . __('user.role.editor') . " $e)",
-					//'editor_url' . $e => __('user.url') . " (" . __('user.role.editor') . " $e)",
-					//'editor_biography' . $e => __('user.biography') . " (" . __('user.role.editor') . " $e)"
-			));
-			for ($d = 1; $d <= $maxDecisions; $d++) {
-				$columns = array_merge($columns, array(
-						'editor_decision' . $e . $d => __('submission.editorDecision') . " $d " . " (" . __('user.role.editor') . " $e)",
-						'editor_decision_date' . $e . $d => __('common.dateDecided') . " $d " . " (" . __('user.role.editor') . " $e)"
-				));
+			// Load section title information
+			$sectionId = $publication->getData('sectionId');
+			if (!isset($sectionTitles[$sectionId])) {
+				$section = $sectionDao->getById($sectionId);
+				$sectionTitles[$sectionId] = $section->getLocalizedTitle();
 			}
+
+			// Store the submission results
+			$results[] = [
+				'submissionId' => $submission->getId(),
+				'title' => $publication->getLocalizedTitle(),
+				'abstract' => html_entity_decode(strip_tags($publication->getLocalizedData('abstract'))),
+				'authors' => array_map(function($author) {
+					return [
+						$author->getLocalizedGivenName(),
+						$author->getLocalizedFamilyName(),
+						$author->getData('orcid'),
+						$author->getData('country'),
+						$author->getLocalizedData('affiliation'),
+						$author->getData('email'),
+						$author->getData('url'),
+						html_entity_decode(strip_tags($author->getLocalizedData('biography'))),
+					];
+				}, $publication->getData('authors')),
+				'sectionTitle' => $sectionTitles[$sectionId],
+				'language' => $publication->getData('locale'),
+				'coverage' => $publication->getLocalizedData('coverage'),
+				'rights' => $publication->getLocalizedData('rights'),
+				'source' => $publication->getLocalizedData('source'),
+				'subjects' => join(', ', $submissionSubjectDao->getSubjects($submission->getCurrentPublication()->getId(), array($submission->getLocale()))[$submission->getLocale()]??[]),
+				'type' => $publication->getLocalizedData('type'),
+				'disciplines' => join(', ', $submissionDisciplineDao->getDisciplines($submission->getCurrentPublication()->getId(), array($submission->getLocale()))[$submission->getLocale()]??[]),
+				'keywords' => join(', ', $submissionKeywordDao->getKeywords($submission->getCurrentPublication()->getId(), array($submission->getLocale()))[$submission->getLocale()]??[]),
+				'agencies' => join(', ', $submissionAgencyDao->getAgencies($submission->getCurrentPublication()->getId(), array($submission->getLocale()))[$submission->getLocale()]??[]),
+				'status' => $submission->getStatus() == STATUS_QUEUED ? $this->getStageLabel($submission->getStageId()) : __($statusMap[$submission->getStatus()]),
+				'url' => $request->url(null, 'workflow', 'access', $submission->getId()),
```

Comment:
I think this should be `$submission->getBestId()`

Reply:
Is that the case even for workflow URLs now?

Reply:
D'oh! My mistake. Paying close attention to the wrong thing! :joy:

Reply:
Not a problem, I almost made the same change myself!